scripts/gen_reports.py

```diff
@@ -121,20 +138,24 @@ def generate_org_report_files(orgs):
             "repo_owner": org.login
         }
 
-        org_metric_table_headings = [
-            'commits_count',
-            'issues_count',
-            'open_issues_count',
-            'closed_issues_count',
-            'pull_requests_count',
-            'open_pull_requests_count',
-            'merged_pull_requests_count',
-            'closed_pull_requests_count',
-            'forks_count',
-            'stargazers_count',
-            'watchers_count',
-            'followers_count'
-        ]
+        #Define headings as key value pairs where
+        #   key -> value = heading -> desired_behavior
```

Comment:
@decause-gov Note the desired behavior for each value and whether or not they seem correct